pkg/api/util/matrix/matrix_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package quota
+package matrix
```

Comment:
sgtm

On Tue, Aug 16, 2016 at 10:18 AM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/util/matrix/matrix_test.go
> <https://github.com/kubernetes/kubernetes/pull/30152#discussion_r74979105>
> :
>
> > @@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
> >  limitations under the License.
> >  */
> >
> > -package quota
> > +package matrix
>
> resources sounds good. @derekwaynecarr <https://github.com/derekwaynecarr>,
> @vishh <https://github.com/vishh> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30152/files/3fe5f6b4f2ff77e2c5cab405a8e8ec46ca92b15e#r74979105>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGvIKN-Wk1VRS61uN9ls2oN9aWqLF7tCks5qgfD2gaJpZM4Jd6Cu>
> .
>